test/e2e/testdata/prometheus.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
also being created in https://github.com/envoyproxy/gateway/pull/2108

Reply:
this one `LoadBalancer` type